money-rails.gemspec

```diff
@@ -26,8 +26,8 @@ Gem::Specification.new do |s|
 
   s.require_path = "lib"
 
-  s.add_dependency "money",         "~> 6.13.2"
-  s.add_dependency "monetize",      "~> 1.9.0"
+  s.add_dependency "money",         "~> 6.14"
+  s.add_dependency "monetize",      "~> 1.11"
```

Comment:
@antstorm any issue with locking to a less rigid version here?

Reply:
None from me! What do you think of `< 2.0`?

Reply:
Also should `money` be loosened to `< 7.0` as well?

Reply:
it should be okay, but it would be a change of policy. I am cool with it if @antstorm cannot think of any adverse consequences